.github/workflows/publishVSCode.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - run: rm ./SHA256
       - run: |
               git add SHA256.md
-              git commit -m "chore: updated SHA256 v${{ inputs.version }}"
+              git commit -m "chore: updated SHA256 v${{ inputs.version }} [skip ci]"
```

Comment:
But can run read inputs?!

Reply:
ha good catch...must not have burned it yet